client/js/main.js

```diff
@@ -137,7 +139,11 @@ function getDocsFromBertha (docs, republish = false) {
 			}
 
 			// Set the options globally
-			parseOptions(config);
+			const sheetTitle = parseOptions(config).title || undefined;
```

Comment:
parseOptions also overrides options values to those set from the sheet (if present) and it just returns the global options object.

This seems to be the equivalent of `sheetTitle = config.title || options.title || undefined`

So if a title gets set in the queryParams you'll end up with all of the titles being set to that value.